bpf-gpl/tc.c

```diff
@@ -699,23 +710,14 @@ static CALI_BPF_INLINE struct fwd calico_tc_skb_accepted(struct __sk_buff *skb,
 		seen_mark = CALI_SKB_MARK_SEEN;
 	}
 
-
-	struct tcphdr *tcp_header = (void*)(ip_header+1);
-	struct udphdr *udp_header = (void*)(ip_header+1);
-
-	__u8 ihl = ip_header->ihl * 4;
-
-	size_t l4_csum_off = 0, l3_csum_off;
-	int res = 0;
-	bool encap_needed = false;
-	uint32_t fib_flags = 0;
-
 	/* XXX we cannot pass the related ICMP after NATing back yet, so we need
 	 * to act here, we know we are forwarding.
 	 */
+
+	/* XXX XXX XXX */
```

Comment:
?

Reply:
thanks for reminding me of this. Fixed the comment. I decided that it is easier to handle it in here than to let the host to handle it. At least for now.